### ISSUE TITLE
I2G DATA

### ISSUE THREAD

**Issue by q104769424** [jtchen0528/PCL-I2G#7]
Hi, can you share I2G dataset from FF++, because I meet some errors, but I only want the dataset to train my model, thanks. email is q104769424@gmail.com

**Comment by jhaotingc** [comment]
I don't think that I can just share FF++ data to anyone. Sharing these deepfake photos has some ethical issue. That's why we need to fill out some forms when accessing to datasets. And this is probably why the original authors did not release the code and training dataset.
